evansmusomi/burger-builder
# Improves UX and Codebase

*What this does*
- Fixes redirect to homepage
- Enables side drawer to automatically close when clicked
- Ensures burger preview is always centered
- Uses updateObject utility across the app

*How to verify*
- A user who signs up to order should be redirected to the checkout page
- A user should see the side drawer close when they select a nav item on it
- A user should always see the burger preview centered on the screen
- A developer should be able to confirm that reducers and components updating state use the shared utility
